### PR TITLE
 japanese translation

### PR DESCRIPTION
**Pull request by ghost**
Changes proposed in this pull request:
-　Add japanese translation

How to test the feature manually:
I tested it on Docker

Pull request checklist:
- [ ✔] clear commit messages
- [ ✔] code manually tested

This is the result of my translation .

1631005928_grim

### CODE REVIEW

File: app/i18n/ja/admin.php [hunk 1]
```diff
@@ -0,0 +1,206 @@
+<?php
+
+return array(
+	'auth' => array(
+		'allow_anonymous' => '標準のユーザーの記事が匿名のユーザーでも読めるようにします。\' (%s)',	// TODO - Translation
```
Comment (Frenzie) [inline app/i18n/ja/admin.php]: Not super important I suppose, but these don't look very todo. ^_^

Comment (ghost) [reply app/i18n/ja/admin.php]: Thank you for pointing out the issue!
I remove ` // TODO - Translation `

**Review by Frenzie — APPROVED** [review]

**Comment by Alkarex** [comment]
@312k Many thanks for the new translation! Please see a few changes https://github.com/FreshRSS/FreshRSS/pull/3828/commits/f85fb9e0db694541c8abbf9e88153423f60c254b
See in particular the remaining TODOs, whether you would like to translate some of them. Otherwise it is fine as it is.

**Comment by ghost** [comment]
@Alkarex 
Thank you your changes!
I fixed it and fix spelling discrepancies.

**Comment by Alkarex** [comment]
@312k Thank you! Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/edge/CREDITS.md

**Pull request merged**